### ISSUE TITLE
Integrate summaries into individual pages

### ISSUE THREAD

**Issue by sbddesign** [atlantabitdevs/bitdevs-upgrade#19]
When generating the individual page for a given markdown file (currently being worked on in #18) we need to be able to check if a summary file exists in the `/sumaries` file corresponding to that markdown file. If so, we need to be able to pull in the content from that markdown file.

For this issue, don't worry about making it look pretty or whatever. Just focus on finding the JSON summary and pulling it into the page. If the JSON is pulled in as a javascript object, frontend can iterate to make it look nicer.

**Comment by niteshbalusu11** [comment]
There is a sample summary on the master branch in the summaries dir. 
Summaries are json files 
that have this format 
summary-yyyy-mm-dd.json

Summaries object has 
Title
Summary 
Explain like I am 15 sunmary

pick up these files and render them as static pages in nextjs. Maybe a dynamic route.

**Comment by sbddesign** [comment]
What needs to happen here:
- Inside `components/SummaryLiinktsx`
- Use the `props.slug` to find a corresponding JSON file
- Inside that JSON, find a corresponding entry that matches the `props.href`
- Inject the cliff notes and ELI5 into their respective places in the HTML

**Comment by sbddesign** [comment]
@niteshbalusu11 finished it!
